src/main/scala/org/scalacheck/Test.scala

```diff
@@ -225,7 +225,7 @@ object Test {
     ) throw new IllegalArgumentException("Invalid test parameters")
   }
 
-  private[scalacheck] lazy val cmdLineParser = new CmdLineParser {
+  private[scalacheck] object cmdLineParser extends CmdLineParser {
```

Comment:
Should probably be renamed `CmdLineParser` per naming conventions elsewhere for class level objects in the code base, even if _uniform access_ would suggest otherwise.  This requires renaming a bunch of references elsewhere, but that probably helps explain why the`reflectiveCalls` import is removed in those files.

Reply:
Done!